app/src/main/java/com/dessalines/thumbkey/keyboards/FrenchNumeric.kt

```diff
@@ -0,0 +1,338 @@
+package com.dessalines.thumbkey.keyboards
+
+import com.dessalines.thumbkey.utils.ColorVariant
+import com.dessalines.thumbkey.utils.FontSizeVariant
+import com.dessalines.thumbkey.utils.KeyAction
+import com.dessalines.thumbkey.utils.KeyC
+import com.dessalines.thumbkey.utils.KeyDisplay
+import com.dessalines.thumbkey.utils.KeyItemC
+import com.dessalines.thumbkey.utils.KeyboardC
+import com.dessalines.thumbkey.utils.SwipeDirection
+import com.dessalines.thumbkey.utils.getLocalCurrency
+
+val FRENCH_NUMERIC_KEYBOARD =
```

Comment:
There's probably a way to do this via `.copy` of the other numeric, but someone could always do that later.